api/features/import_export/views.py

```diff
@@ -74,6 +76,37 @@ def download_feature_export(request: Request, feature_export_id: int) -> Respons
     return response
 
 
+@swagger_auto_schema(
+    method="GET",
+    responses={200: "Flagsmith on Flagsmith File downloaded"},
+    operation_description="This endpoint is to download an feature export to enable flagsmith on flagsmith",
+)
+@api_view(["GET"])
+@permission_classes([permissions.AllowAny])
+def download_flagsmith_on_flagsmith(request: Request) -> Response:
+    from django.http import Http404
```

Comment:
Why the local import?

Reply:
I honestly don't recall. Probably a debugging issue. Sorry about that I will update. Thanks for the catch!